assets/src/scripts/components/map.js

```diff
@@ -0,0 +1,22 @@
+const HYDRO_URL = 'https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Esri_Hydro_Reference_Overlay/MapServer';
+
+
+function attachToNode(node, {latitude, longitude, zoom}) {
```

Comment:
There will be cases (maybe @jkreft-usgs  can find one for you) where latitude/longitude will be the null. In those cases, I'm assuming we don't display the map.